[2025-03-11T09:18:08.476+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-11T09:18:08.816+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_all_tables.orders manual__2025-03-11T09:17:10.388824+00:00 [queued]>
[2025-03-11T09:18:08.935+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_all_tables.orders manual__2025-03-11T09:17:10.388824+00:00 [queued]>
[2025-03-11T09:18:09.026+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-11T09:18:09.209+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): orders> on 2025-03-11 09:17:10.388824+00:00
[2025-03-11T09:18:09.275+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=8323) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-11T09:18:09.300+0000] {standard_task_runner.py:72} INFO - Started process 8333 to run task
[2025-03-11T09:18:09.302+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_all_tables', 'orders', 'manual__2025-03-11T09:17:10.388824+00:00', '--job-id', '235', '--raw', '--subdir', 'DAGS_FOLDER/validate_all_tables_dag.py', '--cfg-path', '/tmp/tmp3iic_9k1']
[2025-03-11T09:18:09.380+0000] {standard_task_runner.py:105} INFO - Job 235: Subtask orders
[2025-03-11T09:18:09.814+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_all_tables.orders manual__2025-03-11T09:17:10.388824+00:00 [running]> on host efacfdd0be48
[2025-03-11T09:18:12.623+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_all_tables' AIRFLOW_CTX_TASK_ID='orders' AIRFLOW_CTX_EXECUTION_DATE='2025-03-11T09:17:10.388824+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-11T09:17:10.388824+00:00'
[2025-03-11T09:18:12.748+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-11T09:18:13.144+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-11T09:18:13.353+0000] {logging_mixin.py:190} INFO - Current task name:orders state:running start_date:2025-03-11 09:18:08.834512+00:00
[2025-03-11T09:18:13.481+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables and current dag run status:running
[2025-03-11T09:18:13.639+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-11T09:18:15.591+0000] {logging_mixin.py:190} INFO - Validation successful. Found 1615 order records.
[2025-03-11T09:18:15.620+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-11T09:18:15.726+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-11T09:18:15.752+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=validate_all_tables, task_id=orders, run_id=manual__2025-03-11T09:17:10.388824+00:00, execution_date=20250311T091710, start_date=20250311T091808, end_date=20250311T091815
[2025-03-11T09:18:16.139+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-11T09:18:16.141+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-11T09:18:16.150+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables queued_at:2025-03-11 09:17:10.479662+00:00
[2025-03-11T09:18:16.151+0000] {logging_mixin.py:190} INFO - Task hostname:efacfdd0be48 operator:PythonOperator
[2025-03-11T09:18:16.309+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-11T09:18:16.401+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
